src/authenticationMiddleware.ts

```diff
@@ -18,11 +23,17 @@ export const authenticationMiddleware = (opts: Options) => async (
   res: Response,
   next: NextFunction,
 ) => {
-  required(opts.iapProjectNumber, "iapProjectNumber");
-  required(opts.iapServiceID, "iapServiceID");
+  if (opts.audience) {
+    if (!(typeof opts.audience === 'string' && opts.audience.match(audienceRegex))) {
```

Comment:
This should be required. I'd rather create a deprecated function which wraps the deprecated fields and add a check after required for the correct regex format.